spinalcordtoolbox/scripts/sct_compute_ascor.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+#
+# Compute adapted Spinal Cord Occupation Ratio (aSCOR) from spinal cord and spinal canal masks.
+# morphometrics.
+#
+# Copyright (c) 2023 Polytechnique Montreal <www.neuro.polymtl.ca>
```

Comment:
I guess we can change this to 2025? :laughing:

Reply:
done here https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/5007/commits/5f412171d907f5b826d7d23c55a6437a8528fa78